scripts/e2e-tests.sh

```diff
@@ -56,15 +56,8 @@ function teardown_test_resources() {
     teardown
   fi
 
-  # Delete Knative Serving images when using boskos.
-  if (( IS_BOSKOS )); then
-    echo "Images in ${DOCKER_REPO_OVERRIDE}:"
```

Comment:
as long as it's not knative releases folder, it's still ok to delete, right? we don't have to block delete for all cases